python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -1092,6 +1092,30 @@ def run(
         else:
             self._summarize(result.exposure, summary_stat_catalog, result.background)
 
+        # Output post-subtraction background stats to task metadata:
+        bgIgnoreMaskPlanes = [
```

Comment:
Please add a comment describing what "ignore" means here - pixels with any of these masks are excluded from the background-related statistics metric calculations. I get worried when I see "badMaskPlanes" or "ignoreMaskPlanes" out of the blue without an explicit reassurance this is just for metrics.

Also, consider making this configurable vs hard wired, in the event we want to (I don't know) add STREAK to it some day...

Reply:
Isn't the new SPIKE mask also available now, via calibrateImage? Should it be in the list?

Reply:
Thanks for hopping on this so quickly. 👍 to adding the comment and config. Not sure about SPIKE. I did have that, but then it broke pipe_task tests in Jenkins. I've no idea why, cos it should exist.